app/views/admin/slides/edit.html.erb

```diff
@@ -0,0 +1,3 @@
+<h3><%= t(:edit_slide) %>&nbsp;&nbsp;<small><a href="/admin/dashboards/index"><%= t :back_to_dashboard %></a></small></h3>
+
+<%= render :partial => 'slide_edit_form', :locals => { :slide => @slide } %>
```

Comment:
__[rails_best_practices]__
* simplify render in views [:ref](http://rails-bestpractices.com/posts/2010/12/04/simplify-render-in-views/)

<!-- Comment by [SideCI](https://sideci.com) -->